app/serializers/track_serializer.rb

```diff
@@ -32,4 +32,9 @@ def bitrate
   def location_id
     object.audio_file&.location_id
   end
+
+  def play_count
+    # This fallback will only be accessed if the track was just created
```

Comment:
If so, setting the attribute in the track constructor seems cleaner to me.

Reply:
It would be cleaner, but `play_count` depends on the `current_user`, which we cannot access in the model.

Reply:
I don't really get your comment? You don't use the `current_user` here either? I just mean that the fallback value should be set in the constructor, not that the value should be calculated in the constructor.